virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -72,6 +72,23 @@ The SVVP Certification applies to:
 
 //CNV-28720 Release note: CHANGE
 
+//CNV-28724 Release note: storage profile defaults
+* `claimPropertySets` are now populated by default in storage profiles for the following additional provisioners:
+
+** Dell PowerFlex
+** Dell PowerScale
+** Dell Unity
+** Dell PowerStore
+** Hitachi Virtual Storage Platform (VSP)
+** IBM Fusion Hyper-Converged Infrastructure (HCI)
+** IBM Fusion General Parallel File System (GPFS2)
+** IBM Fusion Software-Defined Storage (SDS/ODF) 
+** IBM Fusion block arrays
+** Hewlett Packard Enterprise 3PAR
+** Hewlett Packard Enterprise Nimble
+** Hewlett Packard Enterprise Alletra
+** Hewlett Packard Enterprise Primera
```

Comment:
The only HPE provisioner added is csi.hpe.com, so not sure it covers all of these.